docker-compose.yml

```diff
@@ -179,6 +179,7 @@ services:
     volumes:
       - elasticsearch-data:/usr/share/elasticsearch/data
       - elasticsearch-plugins:/usr/share/elasticsearch/plugins
+      - ./elasticsearch/config:/usr/share/elasticsearch/config
```

Comment:
Is the ./ needed? Ifso why isn't it added to the top 2?
And why not use elasticsearch-config like the others?

Reply:
Relative versus named paths, that's why the `./` is needed. It's an actual folder on the system, instead of some named volume "somewhere" in docker's environment.